toonz/sources/common/trop/brush.cpp

```diff
@@ -1,4 +1,7 @@
 
+#ifdef LINUX
+#include <cstring>
```

Comment:
For some reason it actually built without it when using -fno-implicit-templates but then it wouldn't link, and removing it made it appear as an error. Maybe it's just my current headers (I'm using Debian Sid), I'll try to do an upgrade and see if it's still needed.